Tests/SotoCoreTests/WaiterTests.swift

```diff
@@ -0,0 +1,325 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Soto for AWS open source project
+//
+// Copyright (c) 2017-2021 the Soto project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Soto project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import AsyncHTTPClient
+import NIO
+import SotoCore
+import SotoTestUtils
+import XCTest
+
+class WaiterTests: XCTestCase {
+    static var awsServer: AWSTestServer!
```

Comment:
Out of interest why are these static properties instead of instance properties? Won't that stop you being able to run the tests in parallel?

Reply:
Yeah I guess so. I've never really run the tests in parallel. Will change